lib/src/source/hosted.dart

```diff
@@ -610,6 +666,29 @@ class HostedSource extends CachedSource {
     return null;
   }
 
+  Future<void> _cacheAdvisoriesResponse(
+    Map<String, dynamic> body,
+    PackageRef ref,
+    SystemCache cache,
+  ) async {
+    final path = _advisoriesCachePath(ref, cache);
+    try {
+      ensureDir(p.dirname(path));
+
+      await writeTextFileAsync(
+        path,
+        jsonEncode(
+          <String, dynamic>{
+            ...body,
```

Comment:
Maybe we want to at a timestamp here?
```suggestion
            ...body,
            'fechedAt': DateTime.now.[convert to string...]
```